### ISSUE TITLE
console.log is not meant to be used on prod

### ISSUE THREAD

**Issue by eugef** [singingwolfboy/react-mailmerge#9]
`console.log` is synchronous and should be avoided in production - here is an explanation why: https://nodejs.org/docs/latest-v6.x/api/process.html#process_a_note_on_process_i_o

Probably getting rid of console.log might help to fix unexpected slow-downs on Lambdas.

**Comment by singingwolfboy** [comment]
Do you have a suggestion for how to do logging properly on Lambda?

**Comment by eugef** [comment]
I am not very familiar with Lambda specifics, but maybe this solution can work for you http://www.daveeddy.com/2012/12/06/high-performance-nodejs-logging-with-consolelog-buffering/

**Comment by eugef** [comment]
Other option would be to try pino https://github.com/pinojs/pino

**Comment by singingwolfboy** [comment]
I don't think that the issue we're seeing on Lambda is related to `console.log`. As I mentioned in [yesterday's presentation](https://youtu.be/TLoF1O6vzgY?t=29m41s), the problem we're seeing is that _after_ all the work in the Lambda function is done, sometimes the function continues to run for the full 5 seconds. It happens rarely, inconsistently, and _after_ all the other work is done, including all the logging. Even if using `console.log` is slow, it wouldn't explain that behavior.
